assets/javascripts/discourse/templates/group/assigned.hbs

```diff
@@ -6,9 +6,9 @@
       </div>
     {{/if}}
     {{#load-more selector=".activity-nav li" action=(action "loadMore")}}
-      {{group-assigned-filter show-avatar=false filter="everyone" routeType=route_type assignment_count=group.assignment_count}}
+      {{group-assigned-filter show-avatar=false filter="everyone" routeType=route_type assignment_count=group.assignment_count router=router}}
```

Comment:
Ideally components should be separated from the application, so it would be best if we didn't pass them the router. Can you pass the necessary variables like `q=q ascending=ascending order=order` instead?

Reply:
Sure, ill make this change

Reply:
Done in the latest commit 😃